tools/crm_resource_runtime.c

```diff
@@ -1755,13 +1705,15 @@ cli_resource_execute(resource_t *rsc, const char *requested_name,
         int more, lpc, last;
         char *local_copy = NULL;
 
+        rc = op->rc;
+
```

Comment:
In the case where services_action_sync() returns FALSE, the code currently sets rc to op->rc, whereas this would leave it pcmk_ok. Looking at services_action_sync(), most branches set op->rc, but some don't (e.g. action_get_metadata() failures). We should probably fix that someday. :( For the purposes of this change, I'm thinking if services_action_sync() returns FALSE, use op->rc if it's nonzero, or pcmk_err_generic otherwise.

Reply:
Right, I just needed somewhere to put the comment :)